### PR TITLE
Update Grav generator

### PR DESCRIPTION
**Pull request by dfrankland**
Changed with this commit: https://github.com/getgrav/grav/commit/b2cd4db3955871715ca759f288a885ed44d57b2e

### CODE REVIEW

File: src/apps.json [hunk 1]
```diff
@@ -2295,7 +2295,7 @@
 				"PHP"
 			],
 			"meta": {
-				"generator": "Grav(?:\\s([\\d.]+))?\\;version:\\1"
+				"generator": "GravCMS(?:\\s([\\d.]+))?\\;version:\\1"
```
Comment (AliasIO) [inline src/apps.json]: Should we make `Grav(?:CMS)?` in order to match existing installations?

Comment (dfrankland) [inline src/apps.json]: My only reason for not adding `Grav(?:CMS)?` is that Grav hasn't even reached 1.0 and also https://wappalyzer.com/applications/grav shows many websites not actually using Grav the CMS for some reason... However, besides that, I think adding `Grav(?:CMS)?` wouldn't be bad :) What are your thoughts?

Comment (AliasIO) [inline src/apps.json]: Let's not make it optional in that case, the more specific the better.

**Comment by AliasIO** [comment]
Thanks!

**Comment by dfrankland** [comment]
Cheers! :+1:

**Pull request merged**